docs/index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Flow Client Library (FCL)
+---
+<br />
+<p align="center">
+  <h1 align="center"> FCL JS</h1>
+  <p align="center">
+    <i>FCL JS is the best way to connect your dapp to users, their wallets and Flow.</i>
+    <br />
+    <a href="https://docs.onflow.org/fcl-js/"><strong>Read the docs»</strong></a>
+    <br />
+    <br />
+    <a href="https://docs.onflow.org/fcl/tutorials/flow-app-quickstart/">Quickstart</a>
+    ·
+    <a href="https://github.com/onflow/fcl-js/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/onflow/fcl-js/blob/master/CONTRIBUTING.md">Contribute</a>
+
+  </p>
+</p>
+<br />
+<br />
+
+## What is FCL?
+
+The Flow Client Library (FCL) is a package used to interact with user wallets and the Flow blockchain. When using FCL for authentication, dapps are able to support all FCL-compatible wallets on Flow and their users without any custom integrations or changes needed to the dapp code.
+
+FCL was created to make developing JavaScript applications that connect to the Flow blockchain easy and secure. It defines a standardized set of communication patterns between wallets, applications, and users that is used to perform a wide variety of actions for your dapp. FCL also offers a full featured SDK and utilities to interact with the Flow blockchain.
```

Comment:
"full featured SDK" strikes me as a point of confusion, naming-wise. Could we avoid portraying the mental model that "FCL is composed to SDKs and utilities"? not a blocker for this PR but wanted to flag that we might want to address this at some point

Reply:
Yeah initially FCL was named the JS-SDK as its repo name for a long time - some persistent confusion around this that we've had a hard time messaging it properly. Some people might dismiss it without the SDK as they may believe it's browser only or wallet stuff only. Will think about this some more and address it in a different PR. Ty for the feedback!